src/internal/modules/charge-information/lib/charge-elements/mappers.js

```diff
@@ -40,9 +39,10 @@ const source = formValues => ({
 
 const quantities = formValues => {
   const { authorisedAnnualQuantity, billableAnnualQuantity } = formValues;
+
   return {
     authorisedAnnualQuantity: parseFloat(authorisedAnnualQuantity),
-    billableAnnualQuantity: isEmpty(billableAnnualQuantity) ? null : parseFloat(billableAnnualQuantity)
+    billableAnnualQuantity: !billableAnnualQuantity ? null : parseFloat(billableAnnualQuantity)
```

Comment:
Could you revert this please? `isEmpty` is used here because this value can sometimes receive `''` or `null` or `undefined`.

`isEmpty()` allows us to conveniently check across the board, whereas `!` will only check falsy or undefined.

Reply:
Done.